lib/site.jetpack-module.js

```diff
@@ -0,0 +1,75 @@
+
+/**
+ * SiteJetPackModule methods
+ *
+ * @param {String} sid - site id
+ * @param {WPCOM} wpcom - wpcom instance
+ * @return {Null} null
+ */
+class SiteJetPackModule {
+	constructor( id, sid, wpcom ) {
+		if ( ! sid ) {
+			throw new Error( '`site id` is not correctly defined' );
+		}
+
+		if ( ! ( this instanceof SiteJetPackModule ) ) {
+			return new SiteJetPackModule( id, sid, wpcom );
+		}
+
+		this.wpcom = wpcom;
+		this._id = id;
+		this._sid = sid;
+		this.path = `/sites/${this._sid}/jetpack/modules`;
```

Comment:
not sure the standard here or not, but do we pad spaces around `${ variable }` or leave it as is here?